src/hdr-config.cpp

```diff
@@ -250,6 +250,7 @@ namespace librealsense
                 {
                     try {
                         _pre_hdr_exposure = _sensor->get_option(RS2_OPTION_EXPOSURE).query();
+                        set_sequence_index(0.f);
```

Comment:
Should be moved before the previous statement + add explanatory comment

Reply:
Done